tests/testdata/pgkit_test_db.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE accounts (
   id SERIAL PRIMARY KEY,
-  name VARCHAR(255),
+  name VARCHAR(255) NOT NULL,
```

Comment:
This line didn't cause the test to fail.

Reply:
It was the batch INSERT statement that had `created_at` column value in the first record; but no value in the second.